http_job_queue.go

```diff
@@ -58,34 +59,54 @@ func NewHTTPJobQueue(pool *ProcessorPool, jobBoardURL *url.URL, site, providerNa
 
 // Jobs consumes new jobs from job-board
 func (q *HTTPJobQueue) Jobs(ctx gocontext.Context) (outChan <-chan Job, err error) {
+	logger := context.LoggerFromContext(ctx)
+	if q.buildJobChan != nil {
+		return q.buildJobChan, nil
+	}
```

Comment:
Is .Jobs() only called from one goroutine? If not, this looks like it might be a potential race condition.